Global/Angular.gitignore

```diff
@@ -0,0 +1,30 @@
+dist/
+tmp/
+/out-tsc
+
+# dependencies
+node_modules/
+bower_components/
+
+# IDEs and editors
+.idea/
+jsconfig.json
```

Comment:
# IDEs and editors
.idea/
.project
.classpath
.c9/
*.launch
.settings/
*.sublime-workspace

# Copied from automatic generated .gitignore file in angular 16 project